src/PBot/SyncOMatic/DefaultTemplateRepo.cs

```diff
@@ -31,5 +30,22 @@ static DefaultTemplateRepo()
                 Parts = new Parts("Particular/RepoStandards", TreeEntryTargetType.Blob, "master", "src/RepoName.sln.DotSettings")
             });
         }
+
+        public static List<SyncItem> WithLicense(string license)
+        {
+            if (license == null)
+            {
+                return ItemsToSync;
+            }
+
+            var items = new List<SyncItem>(ItemsToSync);
+            var licenseFile = license != "standard" ? $"{license.ToUpper()}.md" : "RPL.md";
+            items.Add(new SyncItem
+            {
+                Target = "LICENSE.md",
```

Comment:
It's true that `LICENSE` is standard, but it means we'd have to change most of our repos. Casual browsing indicates that we mostly use `LICENSE.md`. AFAIK the GitHub license API also recognises `LICENSE.md` and `LICENSE.txt`.